freqtrade/optimize/hyperopt.py

```diff
@@ -282,9 +283,25 @@ def start(self) -> None:
             timerange=timerange
         )
 
+        if not data:
+            logger.critical("No data found. Terminating.")
+            return
+
+        min_date, max_date = get_timeframe(data)
+        # Validate dataframe for missing values (mainly at start and end, as fillup is called)
+        validate_backtest_data(data, min_date, max_date,
+                               timeframe_to_minutes(self.ticker_interval))
+        logger.info(
+            'Backtesting data from %s up to %s (%s days)..',
```

Comment:
"Hyperopting data ..." ? 

i was ... confused by the output while running this

Reply:
Backtest.backtest() is called with these data (by hyperopt, yes), so "Backtesting data"...
I wanted to change it in the same way in backtestesting.py, where the code was taken (now it uses "Measuring data..." there).

Changed it in both hyperopt.py and backtesting.py